prow/life_of_a_prow_job.md

```diff
@@ -25,8 +25,6 @@ kind: ProwJob
 metadata:
```

Comment:
Note: At some point the beginning paragraphs need to become a bulleted list or something. Not a huge fan of the first person perspective either.

Reply:
Agreed.